### PR TITLE
Add Google.Cloud.CloudBuild.V1

### PR DESCRIPTION
**Pull request by jskeet**
This required a local C# namespace addition. If we're happy with this, I'll start pushing that through internal source control.

### CODE REVIEW

**Comment by chrisdunelm** [comment]
Yes, this LGTM.

**Review by amanda-tarafa — APPROVED** [review]
Yep, LGTM. It's a shame about the service name though.

**Comment by jskeet** [comment]
I've just regenerated now that https://github.com/googleapis/googleapis/commit/dd1d2325b189d3bb69c4a6a802185cefbd8e592e has been merged, and there are no differences other than the sha in the synth.metadata. I've rebased that in, and will merge this PR when it's green (and change the title, and remove the Do Not Merge label).

**Pull request merged**
